src/ildasm/dis.cpp

```diff
@@ -315,7 +315,6 @@ BOOL enumEHInfo(const COR_ILMETHOD_SECT_EH* eh, IMDInternalImport *pImport, DWOR
     COR_ILMETHOD_SECT_EH_CLAUSE_FAT ehBuff;
     const COR_ILMETHOD_SECT_EH_CLAUSE_FAT* ehInfo;
     BOOL    fTryInCode = FALSE;
-    if(g_ehInfo) VDELETE(g_ehInfo);
```

Comment:
Found that this memory allocation is overriden during a call to ```strcpy_s``` implemented in PAL. Trying to find out now why this is the case.